### PR TITLE
Fix mobile styling

### PR DESCRIPTION
**Pull request by adrianoapmartins**
Fixed problems with mobile rendering of the pages:

- homepage
- devtoolsporto
- devtoolsportofeedback

Also fixed the hover action on SVGs which now will invert the color (in respect to the current theme).

### CODE REVIEW

File: components/DevToolsPortoLivestream.tsx [hunk 1]
```diff
@@ -65,15 +65,15 @@ const Card = styled.form`
...
   overflow: hidden;
   color: rgb(var(--text));
   margin-top: 3rem;
 
-  ${media('<=tablet')} {
-    padding: 7.5rem 2.5rem;
-  }
+  // ${media('<=tablet')} {
+  //   padding: 7.5rem 2.5rem;
+  // }
```
Comment (ferreiratiago) [inline components/DevToolsPortoLivestream.tsx:76]: Plesase remove this.

File: components/LogoDevToolsPorto.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    </svg>
+
+
```
Comment (ferreiratiago) [inline components/LogoDevToolsPorto.tsx:7]: Remove this empty lines.

File: components/LogoDevToolsPorto.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default function LogoDevToolsPorto({ ...rest }) {
```
Comment (ferreiratiago) [inline components/LogoDevToolsPorto.tsx:1]: Is this parameter being used? If not, please remove it.

File: views/HomePage/ScrollableBlogPosts.tsx [hunk 4]
```diff
@@ -79,12 +79,19 @@ const SwiperContainer = styled.div`
   
   width: 100vw;
   max-width: 225rem;
-  height: 46rem;
+  // height: 46rem;
```
Comment (ferreiratiago) [inline views/HomePage/ScrollableBlogPosts.tsx:82]: Please remove this.

**Review by ferreiratiago — APPROVED** [review]
Minor improvements.

File: components/LogoDevToolsPorto.tsx (hunk 3 above)
Comment (adrianoapmartins) [reply components/LogoDevToolsPorto.tsx:1]: Yes, it's being used in the DevToolsPorto pages

**Pull request merged**